d2bs/kolbot/libs/MuleLogger.js

```diff
@@ -67,17 +67,19 @@ var MuleLogger = {
 	},
 
 	inGameCheck: function () {
+
 		if (getScript("D2BotMuleLog.dbj") && this.LogGame[0] && me.gamename.match(this.LogGame[0], "i")) {
-			print("\xFFc4MuleLogger\xFFc0: Logging items on " + me.name + ".");
-			D2Bot.printToConsole("MuleLogger: Logging items on " + me.name + ".", 7);
+			print("\xFFc4MuleLogger\xFFc0: Logging items on " + me.account + " - " + me.name + ".");
+			D2Bot.printToConsole("MuleLogger: Logging items on " + me.account + " - " + me.name + ".", 7);
+
 			this.logChar();
+			print("\xFFc2IngameTime \xFFc0is set to: \xFFc2" + parseInt(this.IngameTime) + "\xFFc0 sec");
 
-			while ((getTickCount() - me.gamestarttime) < this.IngameTime * 1000) {
-				delay(1000);
+			while ((getTickCount() - me.gamestarttime)/1000 < this.IngameTime) {
```

Comment:
change to `while ((getTickCount() - me.gamestarttime) < this.IngameTime * 1000) {`